purchase_request_to_requisition/models/purchase_request.py

```diff
@@ -20,7 +20,6 @@ class PurchaseRequestLine(models.Model):
     @api.multi
     @api.depends('requisition_lines')
     def _compute_is_editable(self):
-        self.ensure_one()
```

Comment:
Removing the ensure_one is not enough. You have to loop on self as my previous comment suggest it. This is required with the association of  @api.multi and the use of self